nvpy/notes_db.py

```diff
@@ -300,6 +268,46 @@ def __init__(self, config):
 
         self.config = config
 
+        # simplenote only imported if simplenote_sync activated in config-file  
```

Comment:
Quoting from the `README.rst`:

> nvpy is a simplenote-syncing note-taking tool

I developed and tested assuming that the nvPY user uses simplenote synchronization feature. I don't want to complicate source code just to decouple simplenote library dependencies.